src/open_inwoner/openzaak/zgw_imports.py

```diff
@@ -87,17 +94,16 @@ def import_zaaktype_configs() -> list[ZaakTypeConfig]:
 
     this collapses individual ZaakType versions on their identificatie and catalog
     """
-    client = build_catalogi_client()
-    if not client:
-        logger.warning(
-            "Not importing zaaktype configs: could not build Catalogi API client"
-        )
-        return []
+    proxy = MultiZgwClientProxy(build_catalogi_clients())
+    result = proxy.fetch_zaaktypes_no_cache()
 
-    zaak_types = get_configurable_zaaktypes(client)
-    if not zaak_types:
-        return []
+    if result.has_errors:
+        for response in result.failing_responses:
+            logger.exception(
+                "Client %s encountered an exception", exc_info=response.exception
```

Comment:
- See previous comment about log formatting.
- If the import continues (the function doesn't return or raise anything), log message should be the same as above